src/components/Dropdown/V2/Dropdown.Trigger.tsx

```diff
@@ -14,6 +14,7 @@ export interface Props {
   children?: any
   className?: string
   closeDropdown: () => void
+  disabled: boolean
```

Comment:
@ItsJonQ What are you thoughts on adding this disabled state to the Dropdown component?